tests/helper_test.py

```diff
@@ -464,3 +464,29 @@ def test_command_fails(self):
             yield helper.call_cmd_async(['_____better_not_exist'])
         self.assertEqual(cm.exception.exitCode, 1)
         self.assertTrue(cm.exception.stderr)
+
+
+class TestGetEnv(unittest.TestCase):
+    env_name = 'XDG_CONFIG_HOME'
+    default = '~/.config'
+
+    def test_env_not_set(self):
+        with mock.patch.dict('os.environ'):
+            if self.env_name in os.environ:
+                del os.environ[self.env_name]
```

Comment:
`mock.patch.dict` can clear the dict it mocks: `...dict('os.environ', clear=True)`.

Reply:
I decided against clearing the whole dictionary after all, because I feel a completely empty environment would make the test conditions a lot different from all the other tests in the test suite.